### ISSUE TITLE
Accounts not signed

### ISSUE THREAD

**Issue by v0l** [bisq-network/bisq#3508]
Hi,

This is a follow up from https://twitter.com/no_input_found/status/1189236587114172416

Please let me know what info i can provide to test if this is a valid issue.

I have 2 "Legacy Arbitration" tickets, same account used on both, account age 400+ days.

Regards,
Kieran

**Comment by ripcurlx** [comment]
Hi @v0l!
- What payment method is your account and which currency/ies?
- Was there a payout to the buyer in your dispute case?

**Comment by v0l** [comment]
Payment method was: SEPA
Buyer received the full amount.

**Comment by ripcurlx** [comment]
Hmm - that should put you into the signed accounts list (If you are not in the banned filter). The only thing that could be that the arbitrators have not used all their different arbitrator instances to sign those accounts. They switch onion addresses every couple of months. I'll check back with them.

**Comment by ripcurlx** [comment]
When did the dispute happen and what arbitrator (onion address) was involved?

**Comment by v0l** [comment]
Opened 21st August 2019 

```json
"arbitratorNodeAddress": {
    "hostName": "g72ouph23eznvpuz.onion",
    "port": 9999
  },
```

**Comment by ripcurlx** [comment]
Thanks - that is not long ago.

**Comment by pabpas** [comment]
One of my accounts is in a similar situation. Opened on July 6th 2018.
```
"arbitratorNodeAddress": {
    "hostName": "3b7cft5k4ae6a236.onion",
    "port": 9999
  },
```
I also find strange that my limit for this account is 0.25 BTC although it is not signed.

**Comment by ripcurlx** [comment]
> One of my accounts is in a similar situation. Opened on July 6th 2018.
> 
> ```
> "arbitratorNodeAddress": {
>     "hostName": "3b7cft5k4ae6a236.onion",
>     "port": 9999
>   },
> ```
> 
> I also find strange that my limit for this account is 0.25 BTC although it is not signed.

Buy or sell limit?
When was this account created?

**Comment by pabpas** [comment]
Buy limit. The account today is 629 days old.

**Comment by ripcurlx** [comment]
ok than it is because your account is older than 1st of March 2019. For those account there never was a limit.

**Comment by ripcurlx** [comment]
> Opened 21st August 2019
> 
> ```json
> "arbitratorNodeAddress": {
>     "hostName": "g72ouph23eznvpuz.onion",
>     "port": 9999
>   },
> ```

@v0l Could you provide me with the trade id of this dispute?

**Comment by ripcurlx** [comment]
@v0l If you go to the Buy or Sell offer section do you see offers with signed accounts?
<img width="1184" alt="Bildschirmfoto 2019-10-29 um 22 06 37" src="https://user-images.githubusercontent.com/170962/67809145-7b2c8900-fa98-11e9-8a9d-eb9f9fec1707.png">

**Comment by v0l** [comment]
Trade ID: `2e6rb7-47008ef6-c87a-4f5a-a90e-d51dde1366dc-114`

Everything says "Not signed yet" or "N/A"

![image](https://user-images.githubusercontent.com/1172179/67831402-2bda6d00-fb19-11e9-96ef-8dba31b28ec9.png)

**Comment by v0l** [comment]
I have a lot of warnings like this: 
```
Oct-30 16:42:56.125 [JavaFX Application Thread] WARN  b.c.a.w.AccountAgeWitnessService: The peers trade limit is less than the traded amount.
tradeAmount=0.101 BTC
Peers trade limit=0.0625 BTC 
```

I assume this is because of the propagation issue from this tweet? https://twitter.com/bisq_network/status/1189329757856567296

**Comment by ripcurlx** [comment]
> I have a lot of warnings like this:
> 
> ```
> Oct-30 16:42:56.125 [JavaFX Application Thread] WARN  b.c.a.w.AccountAgeWitnessService: The peers trade limit is less than the traded amount.
> tradeAmount=0.101 BTC
> Peers trade limit=0.0625 BTC 
> ```
> 
> I assume this is because of the propagation issue from this tweet? https://twitter.com/bisq_network/status/1189329757856567296

Do you have a trade id for me I could use to verify it locally with the hotfix fix version?

**Comment by v0l** [comment]
> Trade ID: `2e6rb7-47008ef6-c87a-4f5a-a90e-d51dde1`

I ran the release/v1.2.0 branch, looks like its working ok now.

![image](https://user-images.githubusercontent.com/1172179/67850774-0747ba80-fb44-11e9-86b5-d450b27a58eb.png)

My SEPA account shows as signed now too.

**Comment by ripcurlx** [comment]
> > Trade ID: `2e6rb7-47008ef6-c87a-4f5a-a90e-d51dde1`
> 
> I ran the release/v1.2.0 branch, looks like its working ok now.
> 
> ![image](https://user-images.githubusercontent.com/1172179/67850774-0747ba80-fb44-11e9-86b5-d450b27a58eb.png)

I was able to reproduce the problem live. There is indeed a case that shouldn't be as it is right now in v1.2.0. We'll have a look if we want to add the fix in v1.2.1 as well. Thanks for reporting!
